### DISCUSSION TITLE
How to ignore /api/ on vite plugin SSR routing ?

### DISCUSSION THREAD

**Discussion by fortezhuo** [vikejs/vike#667] [Discussion] [answered]
Hi,

How to ignore /api/ routing on vite plugin SSR ? My dev app have some error "URL /api/trpc isn't matching any of your page routes. See https://vite-plugin-ssr.com/routing for more information about routing." while I integrate tRPC

**Comment by brillout** [comment, marked as answer]
Seems like you didn't properly install tRPC.

Curious: why not [Telefunc](https://telefunc.com/)? (I'm its author.)

**Reply by fortezhuo** [reply]
Hi, 
Thanks for reply. I just remove useWss on fastify trpcPlugin and disable fastify websocket. 

I just found about Telefunc.. I will take a look. But it will be great if you can create page about comparison tRPC and Telefunc, and plus point about Telefunc. What's the benefit using Telefunc instead

**Reply by fortezhuo** [reply]
Just add 
```
export const trpcOptions: FastifyTRPCPluginOptions<any> = {
  prefix: '/api/trpc',
  useWSS: false,
  trpcOptions: { router: appRouter, createContext },
}
```
and solve this problem

**Reply by brillout** [reply]
Btw. contribution welcome for a VPS + tRPC example. (I can then create a vite-plugin-ssr.com/tRPC page and link the example there.)

**Reply by fortezhuo** [reply]
I will update this discussion soon after the example ready

**Reply by brillout** [reply]
Neat, looking forward to.

**Reply by fortezhuo** [reply]
@brillout
Hi, sorry for late response.
here : https://github.com/fortezhuo/vite-plugin-ssr-trpc

**Reply by brillout** [reply]
✅ Added: https://vite-plugin-ssr.com/tRPC.
